internal/stats/stats_test.go

```diff
@@ -1,15 +1,16 @@
 package stats
 
 import (
+	"fmt"
 	"testing"
 
 	"github.com/consensys/gnark/backend"
-	"github.com/consensys/gnark/test"
+	"github.com/stretchr/testify/require"
 )
 
 func TestCircuitStatistics(t *testing.T) {
 	const refPath = "latest.stats"
-	assert := test.NewAssert(t)
+	assert := require.New(t)
```

Comment:
Why use require directly instead of assert?

Reply:
before used to import `gnark/test` ; but we don't need this package here, in this context we just do a "normal unit test", no circuit stuff going on.